cmd/cloud_login.go

```diff
@@ -115,6 +134,28 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		newCloudConf.Token = null.StringFrom(vals["Token"])
 	}
 
+	if newCloudConf.Token.Valid {
+		client := cloudapi.NewClient(
+			c.globalState.Logger,
+			newCloudConf.Token.String,
+			consolidatedCurrentConfig.Host.String,
+			consts.Version,
+			consolidatedCurrentConfig.Timeout.TimeDuration(),
+		)
+
+		var res *cloudapi.ValidateTokenResponse
+		res, err = client.ValidateToken()
+		if err != nil {
+			return fmt.Errorf("can't validate the API token: %s", err.Error())
+		}
+
+		if !res.IsValid {
+			return errors.New("your API token is invalid - " +
+				"please, consult the Grafana Cloud k6 documentation for instructions on how to generate a new one:\n" +
+				"https://grafana.com/docs/grafana-cloud/testing/k6/author-run/tokens-and-cli-authentication")
+		}
+	}
+
```

Comment:
nit: I would probably move this to a function `validateToken` to not have to keep adding to the function and add nolint:funlen. 

`consoliudatedCurrentConfig` also seems like a thing only used for this.